src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -385,6 +383,8 @@
   <ItemGroup>
     <Compile Include="$(BclSourcesRoot)\System\Reflection\Assembly.CoreCLR.cs" />
     <Compile Include="$(BclSourcesRoot)\System\Reflection\AssemblyName.cs" />
+    <Compile Include="$(BclSourcesRoot)\System\Reflection\AssemblyNameHelpers.cs" />
```

Comment:
I would recommend to not worry about the sharing for ZBB. Make this work well for CoreCLR, without sharing. We can figure out the sharing with CoreRT in separate PR - it can wait after ZBB.